mysql/src/finagle_clojure/mysql/client.clj

```diff
@@ -190,7 +190,7 @@
   ([^PreparedStatement stmt params]
     (select-stmt stmt params Row->map))
   ([^PreparedStatement stmt params fn1]
-    (let [params (scala/seq->scala-buffer (map value/box params))
+    (let [params (scala/seq->scala-buffer (map value/parameterize params))
```

Comment:
I totally missed that! Yes, that's the better implementation. I'll switch over; sorry about that.